### PR TITLE
[sonic-py-common] Add function to get hostname

### PR DESCRIPTION
**Pull request by renukamanavalan**
Added one more common function to retrieve hostname.

### CODE REVIEW

File: src/sonic-py-common/sonic_py_common/device_info.py [hunk 1]
```diff
@@ -29,6 +29,10 @@
 BACKEND_ASIC_SUB_ROLE = "BackEnd"
 
 
+def get_hostname():
+    return os.uname()[1]
```
Comment (prsunny) [inline src/sonic-py-common/sonic_py_common/device_info.py:33]: Should this be from DEVICE_METADATA? Not sure if we recently had any case of this one different from `os.uname` and/or this uname output change in any kernel version. Just a thought!
`"DEVICE_METADATA": "localhost": "hostname"`

Comment (jleveque) [reply src/sonic-py-common/sonic_py_common/device_info.py:33]: It depends on whether this function will ever be called before the database is ready.

Comment (prsunny) [reply src/sonic-py-common/sonic_py_common/device_info.py:33]: Agree with function. I'm was only referring, if this function should fetch the hostname from DEVICE_METADATA. Fine with this as well.

Comment (jleveque) [reply src/sonic-py-common/sonic_py_common/device_info.py:33]: Technically, the OS host name will not be configured until the Config DB is populated. Thus, maybe we *should* always read it from Config DB here...

Comment (renukamanavalan) [reply src/sonic-py-common/sonic_py_common/device_info.py]: Changed it. 
Thank you guys, for looking at this in-depth.

**Pull request merged**